### PR TITLE
Keybindings: Current Key Sequence is a multiline textbox now; fixed #…

### PR DESCRIPTION
**Pull request by Defman21**
…1797

Signed-off-by: Defman21 <i@defman.me>

### CODE REVIEW

**Comment by Naatan** [comment]
This is quite a large change for what the bug is. Can you explain the rationale?

**Comment by Defman21** [comment]
I've removed the menulist appearance when there are 2+ keybindings for a command, so that's why the change is that big for such a small feature request.

File: src/chrome/komodo/content/keybindings/keybindings.p.xml [hunk 1]
```diff
@@ -608,32 +577,14 @@
                     <![CDATA[
                     try {
                         var numlabels = this.keylabels.length;
-                        var usePopup = numlabels>1;
                         if (numlabels < 2) {
                             if (numlabels == 0) {
-                                this.currentkey.value = this._bundle.GetStringFromName("none.message");
-                                this.currentlabel = '';
+                                this.currentkey.value = "None"; // this._bundle.GetStringFromName("none.message");
```
Comment (Defman21) [inline src/chrome/komodo/content/keybindings/keybindings.p.xml]: For some reason, `this._bundle` was undefined, so I've replaced it with the "None" word.

**Comment by Defman21** [comment]
screenshot_20170505_165253

screenshot_20170505_165216

**Comment by Naatan** [comment]
I gotta be honest; I'm not a fan. This just makes it more confusing, why are there multiple keybindings shown? Do I press one and then the other?

I think a better solution would be to simply show an indicator for how many keybindings there are. Eg. "Command has 2 Keybindings:".

I agree there is definitely a UX issue there, but the solution here just creates new UX issues of its own. I think this needs a better solution, sorry.

**Comment by Defman21** [comment]
No worries :)

**Pull request closed without merging**